src/entrypoints/publishing/router/publishing/pages/service-messages-page/index.tsx

```diff
@@ -62,33 +62,34 @@ const ServiceMessagesPage: FC<Props> = () => {
         </SC.NoMessages>
       )}
       {serviceMessages?.map(
-        ({
-          id,
-          title,
-          message_type,
-          valid_from,
-          valid_to,
-          short_description
-        }) => (
-          <SC.ServiceMessage
-            key={id}
-            severity={Severity[message_type as keyof typeof Severity]}
-          >
-            <SC.Content>
-              <SC.ServiceMessageTitle
-                to={`${PATHNAME_PUBLISHING}/service-messages/${id}`}
-                forwardedAs={RouteLink}
-              >
-                {title}
-              </SC.ServiceMessageTitle>
-              <SC.Date>
-                <Moment format='DD.MM.YYYY HH:mm'>{valid_from}</Moment> -{' '}
-                <Moment format='DD.MM.YYYY HH:mm'>{valid_to}</Moment>
-              </SC.Date>
-              <div>{short_description}</div>
-            </SC.Content>
-          </SC.ServiceMessage>
-        )
+        ({ id, attributes }) =>
```

Comment:
```suggestion
        ({ id, attributes: {title,
          message_type,
          valid_from,
          valid_to,
          short_description} }) =>
```
Kan man gjøre slik? Er det lurt? :D

Reply:
Fungerer ikke bra her, siden attributes kan være null/undefined. Da begynner typescript å klage.